relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -780,6 +784,13 @@ mod tests {
         "*"
     );
 
+    span_description_test!(
+        span_description_file_extension_with_numbers_only,
+        "backup.2024041101",
```

Comment:
The scrubbing fails on `backup.2024041101.tar.gz` right? Or the simpler `backup-2024041101` 

Seems like we'd need a better approach since the date is just accidentally (?) the last part.

Reply:
I believe `backup.2024041101.tar.gz` would become `*.gz`, because everything before the top-level extension is removed anyway.

Reply:
Ah right, I was practically staring at that code!